app/components/scrollbar/Scrollbar.tsx

```diff
@@ -0,0 +1,168 @@
+import React from "react";
```

Comment:
Can we go with lowercase scrollbar.tsx to match out other files?

Reply:
Done!

Also FYI I got rid of the "events" subject in favor of the usual callback prop approach that you see with most React components (`<HorizontalScrollbar onScroll={...} />`)